back/db/migrate/20241120151945_create_embeddings_similarities.rb

```diff
@@ -0,0 +1,13 @@
+class CreateEmbeddingsSimilarities < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension 'vector'
```

Comment:
Not sure if this is necessary, I think instead we could enable the extension within the Dockerfile?